### ISSUE TITLE
README should be clearer, more quickly

### ISSUE THREAD

**Issue by erikng** [macadmins/nudge#250]
> “Nudge is a ______ that _____ with _____” 

> I had to go to your GitHub and then click the parent project link to view the ReadMe, then scroll half way down to get a real idea of what your software does.

> Nudge doesn't just enforce macOS software updates, it provides a multi-linguistic custom UI with deferrals, which is how it stands out from using JamfPro prompts and regular Bash scripts, etc.

**Comment by securitygeneration** [comment]
I agree, there's a lot of valuable stuff but the legibility is obscure.
